runMolecule.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
I suggest to add some short description of the purpose of the file and how to use it in 'must succes' and 'must fail scenarios'.

Reply:
Nice suggestion, added